lbrynet/core/client/ClientProtocol.py

```diff
@@ -71,17 +69,17 @@ def timeoutConnection(self):
         self.peer.report_down()
         self.transport.abortConnection()
 
-    def connectionLost(self, reason):
+    def connectionLost(self, reason=None):
         log.debug("Connection lost to %s: %s", self.peer, reason)
         self.setTimeout(None)
         self.connection_closed = True
-        if reason.check(error.ConnectionDone):
+        if reason.check(error.ConnectionDone) or reason is None:
```

Comment:
If reason is `None` then `reason.check` will be an attribute error, should be `if reason is None or reason.check(error.ConnectionDone):`